### ISSUE TITLE
Unexpected warning message on new UK Biobank data release

### ISSUE THREAD

**Issue by sritchie73** [sritchie73/ukbnmr#6]
Related to #5 , `remove_technical_variation()` is now returning an unexpected error:

```
Warning message:
In eval(jsub, SDenv, parent.frame()) : NAs introduced by coercion
```

**Comment by sritchie73** [comment]
Stepping through the code line-by-line reveals this is caused by four samples whose "Well.Position.Within.Plate" (UK Biobank field 23660) is labelled with lowercase letters for the row position instead of uppercase letters, e.g. "a06" instead of "A06". 

The warning error thus arises in the code when we split out the row and column position, because it deletes uppercase letters and then converts to integer. Since this doesn't remove the lowercase letters, NAs are introduced by coercion when attempting to convert "a06" to an integer.

These four samples are also not corrected as part of the removal of unwanted technical variation:

(1) when adjusting for well row position, each sample is treated as its own unique group due to the unique row labels ("a", "b", "c", and "e")
(2) when adjusting for column position, these samples are grouped together into an "NA" group

**Comment by sritchie73** [comment]
Fixed in version 2.2.1, Well.Position.Within.Plate now converted to uppercase when extracting from raw field data
